databricks/koalas/internal.py

```diff
@@ -599,34 +605,30 @@ def column_index_names(self) -> Optional[List[str]]:
         """ Return names of the index levels. """
         return self._column_index_names
 
+    def _data_column_names(self):
+        index_columns = set(self.index_columns)
+        data_columns = []
+        for i, (column, idx) in enumerate(zip(self._data_columns, self.column_index)):
+            assert column not in index_columns
```

Comment:
@ueshin, mind if I ask to double check if this is correct?

Reply:
Actually we do have such a case. We can't raise an error.

Reply:
Hmmmm .. okie let me revert it back for now.